src/main/java/oss/fosslight/service/impl/OssServiceImpl.java

```diff
@@ -873,6 +873,11 @@ public String checkExistOssConf(String ossId) {
 	@Override
 	@CacheEvict(value="autocompleteCache", allEntries=true)
 	public String registOssMaster(OssMaster ossMaster) {
+
+		if(ossMaster == null) {
+			return null;

```

Comment:
If oss is null, NPE occurs in the code below

```java
String[] ossNicknames = ossMaster.getOssNicknames(); // Null Pointer Exception
String ossId = ossMaster.getOssId(); // Null Pointer Exception

...

if(ossMaster == null) {
	return null;
}
```

Reply:
I think it doesn't be needed. so I will delete it.

Reply:
There are some cases where the default value of the variable returned by this function is null, and if it is not, it is not a problem because it is wrapped in a try catch.

Reply:
@yugeeklab ,
If you delete this, won't an error occur if you call this function in the case where ossMaster is null?

Reply:
I think so. But there is so many things to do in this project when we think that parameter is null.(And the examine tool also may think parameter is not null.)
So I think It is better to fix it next week after examining the code.